bc_obps/reporting/api/product_data.py

```diff
@@ -0,0 +1,20 @@
+from service.product_service import ProductService
+from .router import router
+from registration.decorators import handle_http_errors
+from django.http import HttpRequest
+from typing import Tuple
+from registration.schema.generic import Message
+from ninja.responses import codes_4xx, codes_5xx
+
+
+##### GET #####
+
+
+@router.get(
```

Comment:
This is something already implemented in registration: `registration/api/v1/regulated_products`
We should reuse / make those endpoints common where we can, instead of maintaining a double implementation

Reply:
@pbastia Fixed this